test/functions/getMetricFunctionsTest.js

```diff
@@ -0,0 +1,1180 @@
+const slug = 'santiment'
```

Comment:
There is not any value in these tests, all of them test a single function `getMetric_`. Passing different metric names won't do any difference because you are faking the response and it always has the same format.